curriculum/challenges/_meta/learn-relational-databases/meta.json

```diff
@@ -2,7 +2,7 @@
   "name": "Learn Relational Databases",
   "isUpcomingChange": true,
   "dashedName": "learn-relational-databases",
-  "order": 2,
+  "order": 1,
```

Comment:
Relational DB cert disappears for me

Reply:
ref https://github.com/freeCodeCamp/freeCodeCamp/pull/44183/commits/50c25328d23bd4cc239e044fec055c424e027254

Reply:
should move that commit to this pr tbh

Reply:
Which branch has that commit?

Reply:
The branch to enable the new curriculum.

Reply:
Added the query fix here so this can land independently.